src/main/java/Duke.java

```diff
@@ -1,10 +1,268 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+
 public class Duke {
+    // Level 6
+    //for pr creation
+    // A-trial run
+    // program to be added
+    // Find out the position of the first '/'
+    public int firstSlashTracker (String string){
+        int location = 0;
+        boolean exist = false;
+        for (int i = 0; i < string.length(); i++){
+            if(string.charAt(i) == '/'){
```

Comment:
Should there be whitespace after "if"? I noticed the same issue in several other places too. 
Reference: "Java reserved words should be followed by a white space." from CS2103 Java coding standard web page

Reply:
Thanks Dahfin, I will change that!